caramel/config.py

```diff
@@ -0,0 +1,85 @@
+#! /usr/bin/env python
+# vim: expandtab shiftwidth=4 softtabstop=4 tabstop=17 filetype=python :
+"""caramel.config is a helper library that standardizes and collects the logic
+ in one place used by the caramel CLI tools/scripts"""
+import argparse
+import logging
+import os
+
+LOG_LEVEL_ENUM = (
+    logging.ERROR,
+    logging.WARNING,
+    logging.INFO,
+    logging.DEBUG,
+)
+
+
+def add_verbosity_argument(parser):
```

Comment:
Why is the shared-common behaviour for all code isolated and make the other code adapt to it?
The pattern of changes introduced here is:

1) "get an argument parser"
2) Add our own argument parsers
3) Extend it with the common defaults
4) parse our own changes
5) perform some common default operatoion

When the code could simply be a:

parser = get_common_parser(....)

Where the common parser would add "verbosity" and "ini file"  settings as "reasonable" defaults.

See exaple below.

Reply:
I disagree, "simple" monolithic implementations can sometimes just be obfuscating I think. In this case the suggestion would compile 3 lines of code to 1 in every tool, but will obfuscate what happens/make it necessary to go into get_common_parser to see what "common" parser options are. I think keeping it separated and modular with descriptive function names will make it easier to understand the code on a first overview.

Reply:
see pull req #62